src/python_service_template/api/health.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+
+router = APIRouter(
+    prefix="/health",
+    tags=["system"],
+)
+
+
+@router.get("")
+async def health_check() -> dict[str, str]:
```

Comment:
healtcheck needs to follow the following structure:

![image](https://github.com/user-attachments/assets/8b9f5ab0-572b-4928-96f7-3ee2af7cea30)